apis/nucleus/src/index.js

```diff
@@ -84,6 +84,7 @@ const DEFAULT_SNAPSHOT_CONFIG = /** @lends SnapshotConfiguration */ {
  * @property {Array<TypeInfo>=} types Visualization types to register
  * @property {Array<ThemeInfo>=} themes Themes to register
  * @property {object=} anything
+ * @property {object=} flags
```

Comment:
I don't think we want to publicly export flags like this, its an internal thing.

Reply:
This is needed to pass apis?.features from qlik-embed to nebula.

Reply:
no, you can override that right? Add a ts-ignore thing.